task/06-conditions-n-loops-tasks.js

```diff
@@ -270,7 +321,13 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
+    const getSum = (num) => num.toString().split('').map(Number).reduce((sum, elem) => sum + elem);
+
+    do {
+        num = getSum(num);
+    } while(num > 9);
+
+    return num;
```

Comment:
Yes, thats what I mean!
But you can do it even shorter)
You can use getDigitalRoot function once again too, instead of making new function inside (because they are the same).
So instead of "do while" you can write just:  if (num > 9) { getDigitalRoot(num) }
Thats it) 
But anyway, good job!